src/pages/Calendar/Calendar.tsx

```diff
@@ -108,6 +108,8 @@ const CalendarPage = ({ eventData }: CalendarPageProps) => {
                   date={event.date}
                   textContent={event.description}
                   galleryArray={event.images}
+                  capacityCurrent={0}
+                  capacityMax={0}
```

Comment:
I can see why you have done this to make it work with historic events but see my comment above as a way to fix this so we can pass the actual data coming from the DB